### ISSUE TITLE
Hangs and crashes since update

### ISSUE THREAD

**Issue by WAEofFishing** [OpenShot/openshot-qt#1908]
**Describe the bug**
whenever I am "previewing" a file or "split clip" the playback hangs when trying to play, move the slider or frame advance with the arrow key.  I get a spinning circle or the header bar of the player window shows (not responding).  If I continually ask the software to do things without waiting it will crash.  This started after the last update.

**System Details (please complete the following information):**
 - Operating System / Distro: W10 Build 1803 (OS Build 17134.165)
 - OpenShot Version 2.4.2
- AMD Ryzen
_ 8GB Ram
_ SSHD (where files are stored)

**To Reproduce**
1. Go to 'split clip'
2. Click on ' play'
3. Track advance
4. Playback hangs,, 30s to 2min

**Expected behavior**
i should be able to frame advance back and forth with minimal to no delay

Additional context
when moving files around on the timeline there are delays also. Working with transitions cause a crash 80% of the time.

**Comment by WAEofFishing** [comment]
If I uninstall to re install a fresh copy will it still recognize the projects that I have saved?

**Comment by WAEofFishing** [comment]
you know, the entire thing is glitching for me. I couldn't list all of the lock ups.  I am thinking the whole install is corrupted.

**Comment by UMNVideo** [comment]
I am having the exact same problem you are. I uninstalled the update and re-enstalled the old version.  I was able to see the old projects, but there were glitches.  Another words, you can load your old projects, but they will have issues that you will have to go back and fix.  I am not sure if this solves the system glitching because I was not experiencing the worst of the glitches when I tried this. 

I hope that helps at all, but I agree that it is a system issue.

**Comment by WAEofFishing** [comment]
1. glad to know someone else is seeing opportunities.
2. i reinstalled the program and same thing happen.
3. all aspects of the workflow are impacted and I have not tried anything with the titles or animations.  This is all just regular workflow.
4.  LOVE that something like this exists and is FREE!

**Comment by DylanC** [comment]
@WAEofFishing - Maybe try to delete the preferences folder? I think its located in something like /users/username/roaming on Windows?

**Comment by mlackey99** [comment]
I am new to Openshot, just downloaded a copy (v2.4.2) yesterday.  I am seeing similar problems.

New project, import video, the control buttons in the video preview window work as advertised, can play, pause, skip to start/end, etc.  Works well.  Next, press either left-arrow or right-arrow keys.  Video stops playing, and for a while am able to step the video frame-by-frame.  As long as I don;t hit the keys too fast, it seems to work.  But when I speed up the rate at which I press the keys, or I press and hold either 
of the arrow keys, it's bad news.  The video preview window freezes, the timelines freeze, and nothing works.  I see the infamous "please wait" spinney circle.  If I click on the main window frame controls, minimize, maximize, close, I see "not responding" on the window header.  Not doing anything with effects, transitions, etc. just learning the basics and trying to navigate frame-by-frame through the video.  I have uninstalled and re-installed several times, behavior is easily repeatable, and not hard at all to replicate.  For me, just trying to nav frame-by-frame is enough to cause the problem.

Running on an HP laptop with 16Gb, Core i7 and Windows 8.1 64-bit.  Will be glad to supply more info if needed.

My goal is simple:  take a 2-hour video (of my son playing at a local club) and from that single video create multiple videos, one for each song.  First pass no fades in/out, no transitions, just hard cuts.  Any thoughts on alternative software that might work while this bug is being ironed out?

**Comment by melfoo** [comment]
I just installed v2.4.2 on Windows 10 Pro.
Microsoft Surface Pro, Core i5, 8GB 

I can't even start a project. 
I imported a video, add to timeline, set a fade and transition. Then go to preview from the beginning and it immediately crashes out. No error message, just shuts down.
I've tried uninstalling and installing again with no luck.
I initially imported 2 videos to the timeline. Then dropped it down to just one. Also tried a different video with no luck. 

When can this be resolved?
